test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -1,14 +1,35 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import argparse
+import datetime
+import re
 import sys
 import torch
 from torch._C import parse_schema
 
 
+white_list = [
+    ('quantize', datetime.date(2019, 10, 1)),
+    ('q_per_channel_axis', datetime.date(2019, 10, 1)),
+]
+
+
+def white_listed(schema, white_list):
+    for item in white_list:
+        if item[1] < datetime.date.today():
+            continue
+        regexp = re.compile(item[0])
+        if regexp.search(schema.name):
```

Comment:
better to do `match` instead of `search` (and do `.*quant.*`) but it doesn't matter now that much (for future)